DataFormats/Detectors/EMCAL/include/DataFormatsEMCAL/Cluster.h

```diff
@@ -0,0 +1,89 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef ALICEO2_EMCAL_CLUSTER_H_
+#define ALICEO2_EMCAL_CLUSTER_H_
+
+#include <array>
+#include <exception>
+#include <iosfwd>
+#include <string>
+#include <vector>
+#include "CommonDataFormat/TimeStamp.h" 
+#include "MathUtils/Cartesian3D.h"
+
+namespace o2
+{
+
+namespace emcal
+{
+
+/// \class Cluster
+/// \brief EMCAL Cluster
+///
+class Cluster : public o2::dataformats::TimeStamp<Double_t>, public Point3D<Double_t>
```

Comment:
indeed, using double for the position is an overkill here.

Reply:
Fixed